### PR TITLE
Update admin-configforms.png

### PR DESCRIPTION
**Pull request by msbemba**
Changed options in Configure Form Processing page

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3932

File: microsoft-365/contentunderstanding/set-up-content-understanding.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ Admins can use the Microsoft 365 admin center to set up [Microsoft SharePoint Sy
 
 Consider the following before you start:
 
-- Which SharePoint sites will you enable form processing? All of them, some, or select sites?
-- What will you name of your default content center?
+- In Which SharePoint sites will you enable form processing? All of them, some, or select sites?
```
Comment (JohanFreelancer9) [inline microsoft-365/contentunderstanding/set-up-content-understanding.md]:
```suggestion
- In which SharePoint sites will you enable form processing? All of them, some, or select sites?
```

File: microsoft-365/contentunderstanding/set-up-content-understanding.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ Admins can use the Microsoft 365 admin center to set up [Microsoft SharePoint Sy
...
+- What will you name your default content center?
 
 You can change your settings after initial setup in the Microsoft 365 admin center.
 
-Prior to setup, make sure to plan for the best way to set up and configure content understanding in your environment. For example, you need to make considerations about the following names of:
+Prior to setup, make sure to plan for the best way to set up and configure content understanding in your environment. For example, you need to make the following decisions:
 
 - The SharePoint sites that you want to enable form processing - all of them, some, or selected sites
```
Comment (JohanFreelancer9) [inline microsoft-365/contentunderstanding/set-up-content-understanding.md]:
```suggestion
- The SharePoint sites in which you want to enable form processing - all of them, some, or selected sites.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Pull request merged**